### PR TITLE
Update questions and answers

### PR DESCRIPTION
**Pull request by emanuele-em**

### CODE REVIEW

File: vba/vba-quiz.md [hunk 1]
```diff
@@ -812,3 +812,26 @@ If Cars(1).Make="Ford" And Cars (1).MPG > 20
...
+
+#### Q61. In Excel, what is the effect of the two statements below?
+
+```
+Range("b3:f5, c4:d9").Select
+    	Range("b3:f5", "c4:d9").Select
+```
+
+- [ ] If specifies the range that is to be selected after the offset is found
+- [x] it specifies the range of cells in which the contents of the offset are to be written
```
Comment (auchynnikau) [inline vba/vba-quiz.md:835]:
```suggestion
- [x] It specifies the range of cells in which the contents of the offset are to be written
```

File: vba/vba-quiz.md [hunk 2]
```diff
@@ -812,3 +812,26 @@ If Cars(1).Make="Ford" And Cars (1).MPG > 20
...
 - [ ] Variant
+
+#### Q60. What is the purpose of the .Range("A1:E5").Select portion of this statement?
+
+```
+ActiveCell.Offset(3,4).Range("A1:E5").Select
+```
+
+- [ ] If specifies the range that is to be selected after the offset is found
+- [x] it specifies the range of cells in which the contents of the offset are to be written
```
Comment (auchynnikau) [inline vba/vba-quiz.md:823]:
```suggestion
- [x] It specifies the range of cells in which the contents of the offset are to be written
```

File: vba/vba-quiz.md [hunk 3]
```diff
@@ -812,3 +812,26 @@ If Cars(1).Make="Ford" And Cars (1).MPG > 20
 - [x] Type
 - [ ] object
 - [ ] Variant
+
+#### Q60. What is the purpose of the .Range("A1:E5").Select portion of this statement?
```
Comment (Ebazhanov) [inline vba/vba-quiz.md:816]:
```suggestion
#### Q60. What is the purpose of the `.Range("A1:E5").Select` portion of this statement?
```

can be improved in the next PR

**Review by Ebazhanov — APPROVED** [review]
LGTM, thank you @emanuele-em 👍

**Pull request merged**